README.md

```diff
@@ -31,6 +31,15 @@ self.volumeButtonHandler = [JPSVolumeButtonHandler volumeButtonHandlerWithUpBloc
 }];
 ```
 
+To enbable/disable the handler:
+```objective-c
+	// Start
+	[self.volumeButtonHandler startHandler:YES]; 
```

Comment:
Can you amend this to specify that the BOOL param here (disableDefaultVolumeHandler) can be used to have the system keep its own handler, i.e. volume buttons will actually change the volume